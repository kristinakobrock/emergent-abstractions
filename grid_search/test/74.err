  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  8%|#######4                                                                                     | 4/50 [00:00<00:01, 25.80it/s] 14%|#############                                                                                | 7/50 [00:00<00:01, 22.20it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 22.14it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 25.03it/s] 34%|###############################2                                                            | 17/50 [00:00<00:01, 26.45it/s] 40%|####################################8                                                       | 20/50 [00:00<00:01, 26.54it/s] 46%|##########################################3                                                 | 23/50 [00:00<00:00, 27.14it/s] 52%|###############################################8                                            | 26/50 [00:01<00:00, 26.57it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 27.84it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 26.02it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 23.83it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 25.29it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 25.34it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 25.36it/s] 96%|########################################################################################3   | 48/50 [00:01<00:00, 25.25it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.57it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 22.34it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 22.14it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 24.84it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 26.65it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.47it/s]
