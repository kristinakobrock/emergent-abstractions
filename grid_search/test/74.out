# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6863319873809814, "acc": 0.5998892784118652, "length": 3.2498393058776855, "mode": "train", "epoch": 1}
{"loss": 0.6853461265563965, "acc": 0.558398425579071, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6626037955284119, "acc": 0.623113751411438, "length": 2.600041151046753, "mode": "train", "epoch": 2}
{"loss": 0.6694678068161011, "acc": 0.6203124523162842, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6532402038574219, "acc": 0.6272656917572021, "length": 2.0737483501434326, "mode": "train", "epoch": 3}
{"loss": 0.6571668386459351, "acc": 0.619097888469696, "length": 1.7109375, "mode": "test", "epoch": 3}
{"loss": 0.6166711449623108, "acc": 0.6653673052787781, "length": 2.9339029788970947, "mode": "train", "epoch": 4}
{"loss": 0.6038384437561035, "acc": 0.681488037109375, "length": 3.28515625, "mode": "test", "epoch": 4}
{"loss": 0.5710142254829407, "acc": 0.7074679732322693, "length": 3.2741801738739014, "mode": "train", "epoch": 5}
{"loss": 0.6008040904998779, "acc": 0.6912902593612671, "length": 3.3671875, "mode": "test", "epoch": 5}
{"loss": 0.5480902194976807, "acc": 0.7253073453903198, "length": 3.269808053970337, "mode": "train", "epoch": 6}
{"loss": 0.5979794859886169, "acc": 0.6956299543380737, "length": 3.51953125, "mode": "test", "epoch": 6}
{"loss": 0.5326657891273499, "acc": 0.7324892282485962, "length": 3.3214800357818604, "mode": "train", "epoch": 7}
{"loss": 0.5934184789657593, "acc": 0.690173327922821, "length": 3.58984375, "mode": "test", "epoch": 7}
{"loss": 0.5289513468742371, "acc": 0.7332210540771484, "length": 3.347142457962036, "mode": "train", "epoch": 8}
{"loss": 0.5901855230331421, "acc": 0.695544421672821, "length": 3.49609375, "mode": "test", "epoch": 8}
{"loss": 0.5227175951004028, "acc": 0.7331161499023438, "length": 3.3586761951446533, "mode": "train", "epoch": 9}
{"loss": 0.5923095941543579, "acc": 0.7031676769256592, "length": 3.484375, "mode": "test", "epoch": 9}
{"loss": 0.5126708149909973, "acc": 0.7410652041435242, "length": 3.378809928894043, "mode": "train", "epoch": 10}
{"loss": 0.5763193964958191, "acc": 0.7023681998252869, "length": 3.44921875, "mode": "test", "epoch": 10}
{"loss": 0.49823135137557983, "acc": 0.7491251230239868, "length": 3.3738319873809814, "mode": "train", "epoch": 11}
{"loss": 0.5698773264884949, "acc": 0.7070373296737671, "length": 3.47265625, "mode": "test", "epoch": 11}
{"loss": 0.4863604009151459, "acc": 0.7620301246643066, "length": 3.380552053451538, "mode": "train", "epoch": 12}
{"loss": 0.5716321468353271, "acc": 0.7086547613143921, "length": 3.44921875, "mode": "test", "epoch": 12}
{"loss": 0.47018173336982727, "acc": 0.7731077671051025, "length": 3.367389440536499, "mode": "train", "epoch": 13}
{"loss": 0.555641770362854, "acc": 0.7422241568565369, "length": 3.42578125, "mode": "test", "epoch": 13}
{"loss": 0.4528791904449463, "acc": 0.7875404357910156, "length": 3.410479784011841, "mode": "train", "epoch": 14}
{"loss": 0.5806429982185364, "acc": 0.7353760600090027, "length": 3.62890625, "mode": "test", "epoch": 14}
{"loss": 0.4426323175430298, "acc": 0.796114981174469, "length": 3.4562346935272217, "mode": "train", "epoch": 15}
{"loss": 0.567555844783783, "acc": 0.7465759515762329, "length": 3.51171875, "mode": "test", "epoch": 15}
{"loss": 0.42487290501594543, "acc": 0.8040933012962341, "length": 3.4741151332855225, "mode": "train", "epoch": 16}
{"loss": 0.5154612064361572, "acc": 0.76898193359375, "length": 3.50390625, "mode": "test", "epoch": 16}
{"loss": 0.40116629004478455, "acc": 0.8256019949913025, "length": 3.554499387741089, "mode": "train", "epoch": 17}
{"loss": 0.5370765328407288, "acc": 0.781018078327179, "length": 3.74609375, "mode": "test", "epoch": 17}
{"loss": 0.3824808895587921, "acc": 0.8352751731872559, "length": 3.7181427478790283, "mode": "train", "epoch": 18}
{"loss": 0.5081182718276978, "acc": 0.7906310558319092, "length": 3.7890625, "mode": "test", "epoch": 18}
{"loss": 0.3650709092617035, "acc": 0.8437356948852539, "length": 3.7808780670166016, "mode": "train", "epoch": 19}
{"loss": 0.474264532327652, "acc": 0.8007996082305908, "length": 3.93359375, "mode": "test", "epoch": 19}
{"loss": 0.34497666358947754, "acc": 0.8525415658950806, "length": 3.8569726943969727, "mode": "train", "epoch": 20}
{"loss": 0.46167904138565063, "acc": 0.811859130859375, "length": 3.9765625, "mode": "test", "epoch": 20}
{"loss": 0.34476426243782043, "acc": 0.853058934211731, "length": 3.9267637729644775, "mode": "train", "epoch": 21}
{"loss": 0.48637694120407104, "acc": 0.8003662824630737, "length": 3.984375, "mode": "test", "epoch": 21}
{"loss": 0.3346503973007202, "acc": 0.8582659363746643, "length": 3.943361282348633, "mode": "train", "epoch": 22}
{"loss": 0.4536580741405487, "acc": 0.8132811188697815, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.3223761320114136, "acc": 0.8654559850692749, "length": 3.9766030311584473, "mode": "train", "epoch": 23}
{"loss": 0.44045209884643555, "acc": 0.8177734017372131, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.3172912299633026, "acc": 0.8660990595817566, "length": 3.9864602088928223, "mode": "train", "epoch": 24}
{"loss": 0.4588647782802582, "acc": 0.8119140267372131, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.3193592429161072, "acc": 0.8658322095870972, "length": 3.9901998043060303, "mode": "train", "epoch": 25}
{"loss": 0.4642221927642822, "acc": 0.822265625, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.309900164604187, "acc": 0.8708988428115845, "length": 3.992705821990967, "mode": "train", "epoch": 26}
{"loss": 0.4603706896305084, "acc": 0.8294921517372131, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.29802432656288147, "acc": 0.8767400979995728, "length": 3.9937660694122314, "mode": "train", "epoch": 27}
{"loss": 0.445883572101593, "acc": 0.8187499642372131, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.29737165570259094, "acc": 0.8768260478973389, "length": 3.9947311878204346, "mode": "train", "epoch": 28}
{"loss": 0.4369267523288727, "acc": 0.8240233659744263, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.28989720344543457, "acc": 0.8792902827262878, "length": 3.9968631267547607, "mode": "train", "epoch": 29}
{"loss": 0.4409143328666687, "acc": 0.8345703482627869, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.2889898121356964, "acc": 0.8823249936103821, "length": 3.997267007827759, "mode": "train", "epoch": 30}
{"loss": 0.44923320412635803, "acc": 0.8244141936302185, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.28802141547203064, "acc": 0.881754457950592, "length": 3.9980223178863525, "mode": "train", "epoch": 31}
{"loss": 0.4396192133426666, "acc": 0.8308593034744263, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.290659099817276, "acc": 0.8829896450042725, "length": 3.995525598526001, "mode": "train", "epoch": 32}
{"loss": 0.4573245346546173, "acc": 0.8265624642372131, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.2859742343425751, "acc": 0.8845630884170532, "length": 3.9984118938446045, "mode": "train", "epoch": 33}
{"loss": 0.4539990723133087, "acc": 0.8248046636581421, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.2870170772075653, "acc": 0.8860155344009399, "length": 3.993438243865967, "mode": "train", "epoch": 34}
{"loss": 0.448474645614624, "acc": 0.8316406607627869, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.2860479950904846, "acc": 0.8849945664405823, "length": 3.9992563724517822, "mode": "train", "epoch": 35}
{"loss": 0.41478630900382996, "acc": 0.8390625715255737, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.27482062578201294, "acc": 0.8881516456604004, "length": 3.995927333831787, "mode": "train", "epoch": 36}
{"loss": 0.43172621726989746, "acc": 0.8398436903953552, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.26148736476898193, "acc": 0.8960130214691162, "length": 3.9997873306274414, "mode": "train", "epoch": 37}
{"loss": 0.4654773473739624, "acc": 0.8330078125, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.2595050632953644, "acc": 0.8994652628898621, "length": 3.9996871948242188, "mode": "train", "epoch": 38}
{"loss": 0.46040108799934387, "acc": 0.8386718034744263, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.26099643111228943, "acc": 0.8969467282295227, "length": 3.9994142055511475, "mode": "train", "epoch": 39}
{"loss": 0.44590839743614197, "acc": 0.8353515267372131, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.26997435092926025, "acc": 0.894486665725708, "length": 3.999429941177368, "mode": "train", "epoch": 40}
{"loss": 0.4217619299888611, "acc": 0.8414062857627869, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.2596549689769745, "acc": 0.8984752893447876, "length": 3.999897003173828, "mode": "train", "epoch": 41}
{"loss": 0.4217112064361572, "acc": 0.8417968153953552, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.25106382369995117, "acc": 0.9031738042831421, "length": 3.997053623199463, "mode": "train", "epoch": 42}
{"loss": 0.41610854864120483, "acc": 0.8443359732627869, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.24965311586856842, "acc": 0.9064956307411194, "length": 3.9999098777770996, "mode": "train", "epoch": 43}
{"loss": 0.42166668176651, "acc": 0.8541015982627869, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.23711268603801727, "acc": 0.9125194549560547, "length": 3.9997475147247314, "mode": "train", "epoch": 44}
{"loss": 0.44218724966049194, "acc": 0.8480468392372131, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.23632612824440002, "acc": 0.9113401174545288, "length": 3.999417304992676, "mode": "train", "epoch": 45}
{"loss": 0.43378522992134094, "acc": 0.8429685831069946, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.23789818584918976, "acc": 0.9118798971176147, "length": 3.9997589588165283, "mode": "train", "epoch": 46}
{"loss": 0.45571932196617126, "acc": 0.8550781607627869, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.24029982089996338, "acc": 0.9108883738517761, "length": 3.999943494796753, "mode": "train", "epoch": 47}
{"loss": 0.4471668303012848, "acc": 0.8478516340255737, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.24398498237133026, "acc": 0.9089574217796326, "length": 3.9999823570251465, "mode": "train", "epoch": 48}
{"loss": 0.44549760222435, "acc": 0.8472656607627869, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.24433031678199768, "acc": 0.9091308116912842, "length": 3.999786615371704, "mode": "train", "epoch": 49}
{"loss": 0.4562467038631439, "acc": 0.8425780534744263, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.2354794293642044, "acc": 0.9128047823905945, "length": 3.999084949493408, "mode": "train", "epoch": 50}
{"loss": 0.4394699037075043, "acc": 0.8511719107627869, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.23037058115005493, "acc": 0.9157895445823669, "length": 3.9989445209503174, "mode": "train", "epoch": 51}
{"loss": 0.4380784034729004, "acc": 0.8539062142372131, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.22313250601291656, "acc": 0.9190104007720947, "length": 3.999458074569702, "mode": "train", "epoch": 52}
{"loss": 0.4260505735874176, "acc": 0.8609375357627869, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.21596908569335938, "acc": 0.922886312007904, "length": 3.9998207092285156, "mode": "train", "epoch": 53}
{"loss": 0.45575475692749023, "acc": 0.8513671159744263, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.21485204994678497, "acc": 0.9205469489097595, "length": 3.999826192855835, "mode": "train", "epoch": 54}
{"loss": 0.4246545135974884, "acc": 0.8603515625, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.21775881946086884, "acc": 0.9222479462623596, "length": 3.999988555908203, "mode": "train", "epoch": 55}
{"loss": 0.4549008011817932, "acc": 0.8513671159744263, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.21610374748706818, "acc": 0.921895444393158, "length": 3.99996280670166, "mode": "train", "epoch": 56}
{"loss": 0.46587425470352173, "acc": 0.8478516340255737, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.20656706392765045, "acc": 0.926931619644165, "length": 3.999986171722412, "mode": "train", "epoch": 57}
{"loss": 0.43535757064819336, "acc": 0.86328125, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.19661720097064972, "acc": 0.9297972917556763, "length": 3.999547243118286, "mode": "train", "epoch": 58}
{"loss": 0.4417456090450287, "acc": 0.8609375357627869, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.19974744319915771, "acc": 0.9296802282333374, "length": 3.9999194145202637, "mode": "train", "epoch": 59}
{"loss": 0.4127121567726135, "acc": 0.8732422590255737, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.20614148676395416, "acc": 0.9280394911766052, "length": 3.9998302459716797, "mode": "train", "epoch": 60}
{"loss": 0.4125104546546936, "acc": 0.8673828840255737, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7565000653266907
